sdk/iot/modelsrepository/src/constants.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft.
+// Licensed under the MIT license.
+
+const isBrowser = () => { try {return this===self;}catch(e){ return false;}};
+const isNode = () => {try {return this===global;}catch(e){return false;}};
```

Comment:
Please make sure to run `prettier`! Check how our other projects have Prettier configured.